.github/workflows/cbbr_build.yml

```diff
@@ -15,10 +15,22 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./db-cbbr/cbbr_build
+    container:
+      image: nycplanning/build-base:latest
+    env:
```

Comment:
So having env at the container level (which I did originally when moving to using containers as that was what most examples looked like) makes variables inaccessible to `use`d actions (see data library action in pluto input ymls below), but declaring them at the job level as we have in the past works fine for both parts of the job in the container and these other calls. So think we should go back to norm of env at job level